oshmem/mca/sshmem/sshmem_types.h

```diff
@@ -112,9 +113,10 @@ typedef struct map_segment {
     sshmem_mkey_t       *mkeys;          /* includes local segment bases in va_base */
     segment_flag_t       flags;          /* enable/disable flag */
     int                  seg_id;
-    char                 seg_name[OPAL_PATH_MAX];
```

Comment:
I see unlink() and memset two zero. However I do not see where seg_name is set to something and I do not see where it is used. (except in debug prints)